### ISSUE TITLE
Weird Image Generated

### ISSUE THREAD

**Issue by Faustusw** [NVlabs/stylegan2-ada-pytorch#211]
## The workable environment
* cuda10.2
* cudnn7.6.5
* visual studio2019
* pytorch1.10.1+cu102
* torchvisoin0.11.2+cu102
* torchaudio0.10.1+cu102
* python3.8

**Describe the bug**
When I try to run the official command:
python generate.py --outdir=out --trunc=1 --seeds=85,265,297,849 --network=https://nvlabs-fi-cdn.nvidia.com/stylegan2-ada-pytorch/pretrained/metfaces.pkl

It works, but the result performing like this:
![seed0085](https://user-images.githubusercontent.com/71389272/151103008-afd765f7-92ff-444a-9fbd-d2230654c8ca.png)

**Terminal shows these:**
Setting up PyTorch plugin "bias_act_plugin"... Done.
Setting up PyTorch plugin "upfirdn2d_plugin"... Done.
Generating image for seed 85 (1/4) ...
Generating image for seed 265 (2/4) ...
Generating image for seed 297 (3/4) ...
Generating image for seed 849 (4/4) ...

**Environment**
--torch=1.10.1+cu113
--cuda=11.3
--cudnn=8.2.1
--python=3.8
--visual studio2019

**Could you please tell me how can I solve this problem.**

**Comment by JimTaeH** [comment]
I have same problem. Could you tell how to solve this? Thank you
